qiskit/aqua/operators/list_ops/list_op.py

```diff
@@ -327,6 +328,9 @@ def reduce(self) -> OperatorBase:
         reduced_ops = [op.reduce() for op in self.oplist]
         return self.__class__(reduced_ops, coeff=self.coeff)
 
+    def to_circuit(self) -> QuantumCircuit:
```

Comment:
A ListOp contains a list of operators, my feeling if we do this, that this should return a list of circuits. But.... since a ListOp can contain other ListOps then this list of circuits can contain other lists. Hence the return type would be a recursive definition (unless the lists were flattened into one larger single list). Is there a use case you have in mind where this conversion is wanted/useful?

Reply:
@woodsp-ibm it might be useful to see the circuit representation of ComposedOp or TensoredOp, please see the #1089, which requested this feature

Reply:
Ah ok, thx, there is quite a bit being discussed around the Operators and I that issue has slipped my mind and I did not notice your reference. BTW if you prefix that by closes or resolves then the merging of the PR will auto close the issue.
Maybe @Cryoris can comment then on his expectation around this. I can see it for sub-classes since the pieces comprise logically one larger operator - for the base ListOp its not so clear to me

Reply:
thank you for the hint with ‘resolve 
implementation in ListOp works fine for both TensoredOp and ComposedOp (that’s why I moved it to base ListOp)
but I overridden it for SummedOp, because the infinite recursion (that you mention) was happenning there.